.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "flexdll"]
```

Comment:
I'm not familiar with Git submodules.  It is my impression that declaring flexdll as a submodule does not force it to be cloned/fetched automatically with the main repo.  Can you confirm?

What's the reason for declaring the submodule? Just a convenience for developers?